src/esn.calendar.libs/app/services/fullcalendar/planning-render-event.service.spec.js

```diff
@@ -0,0 +1,152 @@
+'use strict';
+
+/* global sinon, chai: false */
+
+const expect = chai.expect;
+
+describe('The calFullCalendarPlanningRenderEventService service', function() {
+  var $q, element, session, calendar, event, tr, tds, a, title, fcTime, calUIAuthorizationService;
+
+  function Element() {
+    this.innerElements = {};
+    this.class = [];
+  }
+
+  Element.prototype.addClass = function(aClass) {
+    this.class.push(aClass);
+  };
+
+  Element.prototype.find = function(aClass) {
+    return this.innerElements[aClass];
+  };
+
+  Element.prototype.prepend = sinon.spy();
```

Comment:
```javascript
Element.prototype = {
  addClass: ...,
  find: ...,
  prepend: ...
}
```

And I think all of the logic to initialize `Element` should be put in `beforeEach` block to avoid polution between test cases

Reply:
@tuanlc Putting it in a `beforeEach` block will result in an error `Illegal constructor`